lib/io.c

```diff
@@ -1224,8 +1225,20 @@ grn_io_seg_map_(grn_ctx *ctx, grn_io *io, uint32_t segno, grn_io_mapinfo *info)
     if (!grn_fileinfo_opened(fi)) {
       char path[PATH_MAX];
       gen_pathname(io->path, path, fno);
+      grn_bool path_exists = GRN_TRUE;
```

Comment:
We don't use "third person singular" form for variable name.
You can use `path_exist`.

Reply:
We define all variables at the top of block as I mentioned before.